logistics_planning_base/models/logistics_schedule.py

```diff
@@ -0,0 +1,83 @@
+# © 2023 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3.0 (https://www.gnu.org/licenses/lgpl-3.0.html)
+
+from odoo import models, fields
+
+TRANSPORT_TYPE = [
+    ('ground', 'Ground'),
+    ('ocean-going', 'Ocean-Going'),
+]
+
+class LogisticsSchedule(models.Model):
+    _name = 'logistics.schedule'
+    _description = 'logistics.schedule'
+
+    name = fields.Char(compute='_compute_name')
+    company_id = fields.Many2one('res.company', string='Company', default=lambda self: self.env.company)
+    state = fields.Selection(
+        selection=[
+            ('draft', 'Draft'),
+            ('ready', 'Ready'),
+            ('done', 'Done'),
+            ('cancel', 'Cancel'),
+        ],
+        default='draft'
+    )
+    type = fields.Selection(
+        selection=[
+            ('input', 'Input'),
+            ('output', 'Output'),
+        ]
+    )
+    stock_move_id = fields.Many2one('stock.move')
+    picking_id = fields.Many2one('stock.picking', related='stock_move_id.picking_id')
```

Comment:
If this field is non-stored, is it possible to search accross it, or grouping by it?